src/templates/k8s/loki/values.yaml

```diff
@@ -0,0 +1,15 @@
+loki:
+    auth_enabled: false
+minio:
+  enabled: true
+write:
+  affinity: {}
```

Comment:
```suggestion
  affinity: []
```

Reply:
Changing to empty list here, L8 and L10 gets rid of those (meaningless) startup errors